amazons3.admin.inc

```diff
@@ -59,6 +59,21 @@ function amazons3_admin() {
     '#default_value' => variable_get('amazons3_cname', 0),
   );
 
+  $schemes = array(
+    'https' => 'https',
+    'http' => 'http'
+  );
+  $form['amazons3_domain_scheme'] = array(
+    '#type' => 'select',
+    '#title' => t('CDN Domain Scheme'),
```

Comment:
And to clarify - what I mean is if I'm new to all of this, I wouldn't know if this refers to the backend S3 call or the URL rendered to clients.